src/shared/services/document-management/document-management.service.spec.ts

```diff
@@ -63,4 +65,51 @@ describe('DocumentManagementService', () => {
         .subscribe(documentMetadata => expect(documentMetadata).toEqual(RESPONSE));
     });
   });
+
+  describe('Media viewer', () => {
```

Comment:
We should have more tests for file names of specific cases. ccd may be allowing only certain types of files but the mediaviewer logic should cover all file types appropriately. The viewer itself, then, may be unable to render some file types, but it should be handled inside the viewer. Our file type extraction logic must not assume certain types only.

Reply:
+1 should test for unexpected file types handling as well

Reply:
ok, so doc extension addresses other types ;)

Reply:
Added more test cases.